aopy/preproc/bmi3d.py

```diff
@@ -504,9 +507,26 @@ def find_laser_stim_times(laser_event_times, laser_event_widths, laser_event_pow
     # Correct the widths and powers with the given thresholds
     corrected_widths, widths_above_thr = validate_measurements(laser_event_widths, adjusted_widths, thr_width)
     corrected_powers, powers_above_thr = validate_measurements(laser_event_powers, adjusted_powers, thr_power)
+
+    if debug:
+        print(f"BMI3D recorded {len(laser_event_times)} stims")
+        print(f"Laser sensor crossed threshold {len(laser_sensor_times)} times")
+        if len(laser_event_times) == len(laser_sensor_times):
+            print(f"Average difference: {np.mean(laser_event_times - laser_sensor_times):.4f} s")
+
+        import matplotlib.pyplot as plt
+        plt.figure()
+        time_before = 0.1
+        time_after = 0.1
+        analog_erp = analysis.calc_erp(ds_data, corrected_times, time_before, time_after, ds_fs)
+        t = 1000*(np.arange(analog_erp.shape[1])/ds_fs - time_before)
```

Comment:
Small thing but it might be useful to add comments detailing the units of variables since is seems like you're switching between seconds and ms.